app/routes/_auth.login.tsx

```diff
@@ -193,8 +203,8 @@ export default function Component() {
                 centerText
                 onChange={(e) => setEmail(e.target.value)}
               />
-
-              <Button variant="primary" buttonStyle="fullyRounded" disabled={processingPasskey || isPending}>
+              <Button variant="outlined" buttonStyle="fullyRounded" icon>
```

Comment:
```suggestion
              <IconButton variant="outlined" buttonStyle="fullyRounded" icon>
```

Reply:
I thought so too last week but it seems like there is actually text in the button as well (not only an icon) 😅